### PR TITLE
Iauf 13:Google Analytics

### PR DESCRIPTION
**Pull request by clal23**
Iauf 13:Google Analytics

### CODE REVIEW

File: modules/jnl_isa_styles/jnl_isa_styles.info [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name = ISA Journal Styles
+description = ISA Journal style module
+core = 7.x
+package = Highwire Journal Templates
+version = 7.x-1.0
+project = jnl_isa_styles
+
+less[sheets][] = css/jnl_isa_styles.css.less
```
Comment (nareshk06) [inline modules/jnl_isa_styles/jnl_isa_styles.info:8]: Add newline after line no 8

**Review by nareshk06 — COMMENTED** [review]
See one minor change, rest seems fine for me.

File: modules/jnl_isa_styles/jnl_isa_styles.info (hunk 1 above)
Comment (clal23) [reply modules/jnl_isa_styles/jnl_isa_styles.info]: I have added new line

**Review by nareshk06 — APPROVED** [review]
seems fine

**Pull request merged**